boards/shields/lcd_par_s035/doc/index.rst

```diff
@@ -0,0 +1,39 @@
+.. _lcd_par_s035:
+
+NXP LCD_PAR_S035 TFT LCD Module
+###############################
+
+Overview
+********
+
+The LCD-PAR-S035 is a 3.5” 480x320 IPS TFT LCD module with wide viewing angle
+and 5-point capacitive touch functionality. The LCD module can be controlled
+through either SPI or parallel (8/16bit) 8080/6800.
+More information about the shield can be found
+at the `LCD-PAR-S035 product page`_.
+
+Requirements
+************
+
+This shield can only be used with FRDM-X evaluation kits with a parallel LCD
```

Comment:
Perhaps note that this is the only interface *currently enabled*? Mentioning this since the shield also has a SPI connector.

Reply:
I have added some verbiage about this.